crates/sui-core/src/authority_aggregator.rs

```diff
@@ -1119,7 +1128,8 @@ where
                                     //
                                     // TODO: currently retryable overload and above overload error look redundant. We want to have a unified
                                     // code path to handle both overload scenarios.
-                                    state.retryable_overloaded_stake += weight;
+                                    state.retryable_overload_info.stake += weight;
+                                    state.retryable_overload_info.requested_retry_after = state.retryable_overload_info.requested_retry_after.max(Duration::from_secs(err.retry_after_secs()));
```

Comment:
just thinking out loud, should we use 67 percentile of all suggested values instead of the highest one? cuz it could be MAX by a byzantine validator.

Reply:
Very good point! For a second, I was thinking the malicious can go both ways: by sending a duration that is too big or too small. But I guess the too big case is more damaging since we already perform exponential backoff in the client.

I changed the code to make the retry_after corresponding to a good quorum threshold of validators with the smallest retry after duration.